modules/update-upgrading-cli.adoc

```diff
@@ -66,7 +66,7 @@ endif::openshift-origin[]
 +
 [NOTE]
 ====
-* If there are no available updates, updates that are supported but not recommended might still be available.
+* If there are no available updates, updates that have known issues might still be available.
```

Comment:
"available" double-use is sticky here (and was before too).  Part of that is ClusterVersion's `availableUpdates`, which holds next-hop options that have no known-risks relevant to the cluster.  In this new world where we're trying to focus on all of these updates being reasonable choices, regardless of the presence of assessed-risks, I'm not sure `availableUpdates` has a concise English translation anymore.  [OTA-1272](https://issues.redhat.com/browse/OTA-1272) is me trying to pitch a new UX that would avoid this distinction, so personally I'm fine with whatever language here in this pull, with the hope that eventually OTA-1272 will allow us to drop this line.

Reply:
should we change to recommended instead of available?

Reply:
This effort is trying to remove "supported but not recommended", and I'd assumed that was because we don't want to apply that only `availableUpdates` are recommended (`availableUpdates` are recommended, but some `Recommended!=True` `conditionalUpdates` might be recommended too?).  But :shrug:, it's hard when "available" and "recommended" show up in the API but are maybe no longer the preferred public wording.  If other folks are fine with "recommended" here, that works for me.

Reply:
looks good to me